NEWS

```diff
@@ -1,3 +1,53 @@
+Version 2.8.2 (2021-07-08)
+==========================
+
+Data updates
+------------
+
+- Updated tzdata version to 2020a. (gh pr #1034)
+
+
+Bugfixes
+--------
+
+- Fixed a bug in the parser where non-``ValueError`` exceptions would be raised
+  during exception handling; this would happen, for example, if an
+  ``IllegalMonthError`` was raised in ``dateutil`` code. Fixed by Mark Bailey.
+  (gh issue #981, pr #987).
+- Fixed the custom ``repr`` for ``dateutil.parser.ParserError``, which was not
+  defined due to an indentation error. (gh issue #991, gh pr #993)
+- Fixed a bug that caused b' prefixes to appear in parse_isodate exception
+  messages.
+- Make `isoparse` raise when trying to parse times with inconsistent use of `:`
+  separator. Reported and fixed by @mariocj89 (gh pr #1125).
+- Fixed tz.gettz() not returning local time when passed an empty string.
+  Reported by @labrys (gh issues #925, #926). Fixed by @ffe4 (gh pr #1024)
+
+
+Documentation changes
+---------------------
+
+- Fix a small typo in dateutil/utils.py.
```

Comment:
PR number or drop this line, I'd say.

In either case move it to the end because it's almost devoid of information compared to the other stuff in the NEWS file.